arangod/Graph/TraverserDocumentCache.cpp

```diff
@@ -149,8 +149,8 @@ void TraverserDocumentCache::insertDocument(
           cache::CachedValue::construct(key, keySize, resVal, resValSize));
 
       if (value) {
-        bool success = _cache->insert(value.get());
-        if (!success) {
+        auto result = _cache->insert(value.get());
+        if (!result.ok()) {
           LOG_TOPIC(DEBUG, Logger::GRAPHS) << "Insert document into cache failed";
```

Comment:
Why is the log topic "GRAPHS" here? The cache isn't limited to storing graph data AFAIK.